nixos/modules/services/monitoring/todesk.nix

```
@@ -0,0 +1,43 @@
+{ config, lib, pkgs, ... }:
+
+let
+
+  cfg = config.services.todesk;
+
+in
+
+{
+
+  ###### interface
+  options = {
+
+    services.todesk.enable = lib.mkEnableOption "ToDesk daemon";
+
+  };
+
+  ###### implementation
+
+  config = lib.mkIf cfg.enable {
+
+    environment.systemPackages = [ pkgs.todesk ];
+
+    systemd.services.todeskd = {
+      description = "ToDesk Daemon Service";
+
+      wantedBy = [ "multi-user.target" ];
+      wants = [ "network-online.target" "nss-lookup.target" ];
```

```suggestion
      wants = [ "nss-lookup.target" ];
```
You  shouldn't need to explcitly "want" network. It should autostart